public/sass/_angular.scss

```diff
@@ -1162,3 +1162,133 @@ div.editor-option label {
     }
   }
 }
+
+gf-form-switch[disabled] {
```

Comment:
Just so I understand this; this file exists to support cases where these classes may be used? And it will be removed with Angular?

Reply:
yes, this file exists to collect styles that are now only needed for angular components. if there are any external plugins that rely on these classes, issues should be raised against those plugins as the intention is this file will be removed in grafana 12 when the rest of angular gets removed.